README.md

```diff
@@ -42,7 +47,7 @@ The bot will (at its next start-up) attempt to detect where Tesseract is install
 If that does not work, please manually override it in the `config.yaml`.
 
 ## Configuration
-If you want to use frequent settings / "set it and forget it", you can do so by editing the `config.yaml` file in the data folder (`%APPDATA%\TFT Bot`).
+If you want to use frequent settings / "set it and forget it", you can do so by editing the `config.yaml` file in the data folder (exe: `%APPDATA%\TFT Bot`, python/source: `.\output\`).

```

Comment:
I'm pretty sure there is a comment about this further down below, but probably fine to have twice for visibility/readability.